targets/targets.json

```diff
@@ -3364,6 +3364,7 @@
             "CRC",
             "TRNG",
             "FLASH",
+            "EMAC",
```

Comment:
You enabled EMAC driver here, but not the macro, effectively causing the board always to cause compilation error.

Is this intentional? Can we instead remove the "EMAC" from the list and specify in the wiki that in order to use the driver, you need the HW modifications and enable `EMAC`?

Reply:
It is intentional to keep EMAC enabled, but this #error should be conditioned by ETHERNET connector support MBED_CONF_TARGET_NETWORK_DEFAULT_INTERFACE_TYPE.
I pushed a fix.